crane-cloud/frontend
# Apply data tables fixes

# Description

This PR works on a few fixes for the front end that has been pointed out.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID
https://trello.com/c/A31LlJij

## How Can This Be Tested?
Pull this branch and run it locally, 
refresh the projects page to see the changes and do the same for the apps list and database list pages.

## Checklist:


# Screenshots
![Screenshot from 2021-07-30 09-22-38](https://user-images.githubusercontent.com/32802973/127614368-7cd684ad-edf8-43dc-85e1-f816c7151e4d.png)
![Screenshot from 2021-07-30 09-41-20](https://user-images.githubusercontent.com/32802973/127614379-4f00dd82-1fc3-4eb2-971a-894005ebc153.png)
![Screenshot from 2021-07-30 09-53-49](https://user-images.githubusercontent.com/32802973/127614384-a1451e19-b66c-42af-ae6e-64bbfc16019b.png)
![Screenshot from 2021-07-30 09-53-53](https://user-images.githubusercontent.com/32802973/127614389-93a2cbe9-45e8-46cb-abf5-e7f9ffafd307.png)
